README.md

```diff
@@ -1,25 +1,45 @@
-# OdinGameMasterAdditions
-
-A mod that enhances the gamemaster mode with several tools and QoL improvements:
+<p align="center">
+    <img src="https://github.com/zen-mod/GME_AR/raw/master/extras/assets/logo/black/GME-Logo.png#gh-light-mode-only" width="512">
+    <img src="https://github.com/zen-mod/GME_AR/raw/master/extras/assets/logo/white/GME-Logo.png#gh-dark-mode-only" width="512">
+</p>
+
+<p align="center">
+    <a href="https://github.com/zen-mod/GME_AR/releases">
+        <img src="https://img.shields.io/github/release/zen-mod/GME_AR.svg?style=flat-square&label=Version" alt="Game Master Version">
+    </a>
+    <a href="https://github.com/zen-mod/GME_AR/issues">
+        <img src="https://img.shields.io/github/issues-raw/zen-mod/GME.svg?style=flat-square&label=Issues" alt="Game Master Enhanced Issues">
+    </a>
+    <a href="https://discord.gg/kN7Jnhr">
+        <img src="https://img.shields.io/discord/364823341506363392.svg?label=Discord&style=flat-square&colorB=7683D5" alt="Zeus Enhanced Discord">
+    </a>
+    <a href="https://github.com/zen-mod/GME_AR/blob/master/LICENSE.md">
+        <img src="https://img.shields.io/badge/License-APL-red.svg?style=flat-square" alt="Game Master Enhanced License">
+    </a>
+</p>
+
+**Game Master Enhanced**, also known as **GME**, is an Arma Reforger mod aimed at improving and expanding the functionality of the Game Master real-time editor.
```

Comment:
Do we want to initially mention somewhere in the readme that this was ODIN and has been renamed and moved? 

Thinking if people currently using ODIN gets confused about it suddenly be called something else or not able to find ODIN anymore? 
Although it might be better and enough to just state that on the workshop page, which is the first page users would see if trying to figure out what the mod is.

Reply:
Good point, I was so focused on adapting from ZEN that I forgot about that xD